ckan/lib/activity_streams_session_extension.py

```diff
@@ -142,6 +144,24 @@ def before_commit(self, session):
         for key, activity in activities.items():
             # Emitting activity
             session.add(activity)
+            if activity.data:
+                try:
+                    obj = activity.data['package']
+                except KeyError:
+                    continue
+
+                data = activity.data.copy()
+                data['real_object_id'] = activity.object_id
+
+                session.add(
+                    Activity(
+                        user_id=activity.user_id,
+                        object_id=obj['owner_org'],
```

Comment:
Depending on your auth configuration, not all datasets always belong to an organization so it may be worth checking if `obj.get('owner_org')`